Dockerfile-debian.template

```diff
@@ -64,6 +64,21 @@ RUN set -ex; \
 # Calculate download URL
 ENV VERSION %%VERSION%%
 ENV URL https://files.phpmyadmin.net/phpMyAdmin/${VERSION}/phpMyAdmin-${VERSION}-all-languages.tar.xz
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL maintainer="The phpMyAdmin Team <developers@phpmyadmin.net>" \
+    org.opencontainers.image.title="Official phpMyAdmin Docker image" \
+    org.opencontainers.image.description="Run phpMyAdmin with Alpine, Apache and PHP FPM." \
+    org.opencontainers.image.authors="The phpMyAdmin Team <developers@phpmyadmin.net>" \
+    org.opencontainers.image.vendor="phpMyAdmin" \
+    org.opencontainers.image.documentation="https://github.com/phpmyadmin/docker#readme" \
+    org.opencontainers.image.licenses="GPL-2.0" \
```

Comment:
https://github.com/phpmyadmin/docker/blob/master/LICENSE is 3.0+, isn't it?

Reply:
"org.opencontainers.image.licenses License(s) under which contained software is distributed"
I hesitated but phpMyAdmin is GPL-2.0

Should I change ?

https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md

Reply:
Oh ok. But therefore the SPDX identifier is deprecated: https://spdx.org/licenses/

Reply:
https://github.com/phpmyadmin/phpmyadmin/blob/master/composer.json#L14

6b63869f1c33fbacdc2b74ea2c3fdb0d0daa04b3

Fixed, thank you !